graylog2-web-interface/src/views/components/dashboard/DashboardsOverview/DashboardActions.tsx

```diff
@@ -144,7 +144,8 @@ const DashboardActions = ({ dashboard, isEvidenceModal = false }: Props) => {
           onClick={() => setShowShareModal(true)}
         />
       )}
-      {!!moreActions.length && isEvidenceModal ? moreActions[0] : <MoreActions>{moreActions}</MoreActions>}
+      {!!moreActions.length && isEvidenceModal ? <MoreActions>{moreActions[0]}</MoreActions> : <MoreActions>{moreActions}</MoreActions>}
```

Comment:
Before we continue with this, can we have a chat why there is so much interdependency between the evidence modal in here? I think we should try to clean this up.